controller/controller.js

```diff
@@ -1,4 +1,4 @@
-const { selectCategories, selectReviews } = require('../model/model.js');
+const { selectCategories, selectReviews, selectUsers } = require('../model/model.js');
```

Comment:
I would suggest maybe separating out your model to different files for different data sets e.g categoriesModel, reviewsModel and so on